.github/workflows/build-notebooks-TEMPLATE.yaml

```diff
@@ -20,6 +20,12 @@ jobs:
   build:
     runs-on: ubuntu-22.04
     env:
+      # Some pieces of code (image pulls for example) in podman consult TMPDIR or default to /var/tmp
+      TMPDIR: /home/runner/.local/share/containers/tmpdir
+      # Use the rootful instance of podman for sharing images with cri-o
+      # https://podman-desktop.io/blog/sharing-podman-images-with-kubernetes-cluster#introduction
+      # https://access.redhat.com/solutions/6986565
+      CONTAINER_HOST: unix:///var/run/podman/podman.sock
```

Comment:
when podman cli is started, it consults this env variable; when it is present, then it connects (as a client) to this socket, so things work pretty much as they would in the docker world

the server can be a systemd service running as root, and if current user has permissions on the socket (it has, because of `0666`), then we run rootful podman